Octokit.Tests.Integration/fixtures/OrganizationsHooksFixture.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Octokit.Tests.Integration.fixtures
+{
+    public class OrganizationsHooksFixture : IDisposable
+    {
+        readonly IGitHubClient _github;
+        readonly OrganizationHook _hook;
+        private string organizationFixture;
+
+        public OrganizationsHooksFixture()
+        {
+            _github = Helper.GetAuthenticatedClient();
+            organizationFixture = Helper.Organization;
+            _hook = CreateHook(_github, organizationFixture);
+        }
+
+
+        public string org { get { return organizationFixture; } }
+
+        public OrganizationHook ExpectedHook { get { return _hook; } }
+
+        public void Dispose()
+        {
+            _github.Organization.Hooks.Delete(organizationFixture,_hook.Id);
+        }
+
+        static OrganizationHook CreateHook(IGitHubClient github, string _organizationFixture)
```

Comment:
I had a look at [RepositoriesHooksFixture](https://github.com/octokit/octokit.net/blob/0e664c23aeb21d25a96c8acf50016c9c5b56b1ce/Octokit.Tests.Integration/fixtures/RepositoriesHooksFixture.cs) but it appears to be "correct" ie class variables are readonly and named with underscores, while method variables do not have underscores.

Must've just been something that happened inadvertently on your side after you copied that class and modified it for Org hooks?